docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -285,11 +358,37 @@ The maximum time (in seconds) a request to the server can take. This can be
 overridden by a larger statement-level timeout setting. The value 0 means no
 timeout.
 
+
 `Follow` (default: `yes`)::
 A boolean value (`yes`|`no` / `true`|`false` / `0`|`1`) controlling if the
 driver will follow HTTP redirects.
 
 
+`Packing` (default: `CBOR`)::
+This value controls which data format to encode the REST content in. Possible
+values are:
+
+* `CBOR`: use the Concise Binary Object Representation format. This is the
+preferred encoding, given its more compact format.
+
+* `JSON`: use the JavaScript Object Notation format. This format is more
+verbose, but easier to read, handful in debugging cases.
```

Comment:
Maybe you meant "useful" (instead of "handful")?

Reply:
Thanks, fixed.